contracts/migrations/Migratable.sol

```diff
@@ -29,17 +29,21 @@ contract Migratable {
    */
   mapping (string => mapping (string => bool)) internal migrated;
 
+  string constant private INITIALIZED_ID = "initialized";
```

Comment:
We're 100% sure that private constants are *not* assigned a slot in the storage, right? I've just checked with Remix and it does not seem to be the case.

Reply:
Experimentally, yes. But I don't think I've seen it documented...

We should probably look into it and push the Solidity team to document it because:

https://github.com/zeppelinos/zos-lib/blob/47c82a93d55fefa2f9067c2ea9313316aeda9083/contracts/upgradeability/UpgradeabilityProxy.sol#L24